### DISCUSSION TITLE
Telemetry

### DISCUSSION THREAD

**Discussion by h2atecnologia** [uNetworking/uWebSockets.js#810] [Q&A]
There is any way to use OpenTelemetry with uWebSockets.js ?

**Comment by e3dio** [comment]
You can use [OpenTelemetry](https://github.com/open-telemetry/opentelemetry-js) with any library you want, there is no auto config you need to set it up. You going to leave this issue open? Are you still confused ?

**Reply by h2atecnologia** [reply]
No. Tks.

**Reply by h2atecnologia** [reply]
Please, close this discussion.
